.ci/ci_job_flags.sh

```diff
@@ -137,6 +140,15 @@ case "${CI_JOB}" in
 	export KUBERNETES="yes"
 	export USE_DEVMAPPER="true"
 	;;
+"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR")
+	# This job only tests containerd + k8s
+	init_ci_flags
+	export CRI_CONTAINERD="yes"
+	export CRI_RUNTIME="containerd"
+	export KATA_HYPERVISOR="cloud-hypervisor"
+	# Export any CC specific environment variables
+	export KATA_BUILD_CC="yes"
+	;;
```

Comment:
I think this is a bad merge? the case branch for `CC_CRI_CONTAINERD_CLOUD_HYPERVISOR` is already there a few lines earlier (and the action is slightly different).

Reply:
That looks better now. Thanks!